lib/sidekiq_publisher.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+# FIXES: uninitialized constant ActiveSupport::LoggerThreadSafeLevel::Logger
+require "logger"
+require "active_record"
```

Comment:
As this seems to be fixing an issue with rails relying on concurrent-ruby requiring one of its own dependencies, what do you think of instead pinning concurrent-ruby to 1.3.4 in the Appraisals for rails versions before 7.1, which would fix the build for this gem without including a fix for something unrelated?

https://github.com/rails/rails/issues/54260

Reply:
I used the fix from [here](https://stackoverflow.com/a/79379493) because I did not want to pin `concurrent-ruby`.

Reply:
I was thinking that concurrent-ruby would only be pinned in the Appraisals, not the gemspec.